app/lib/event_store/services/service_deleted_event.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class EventStore::Services::ServiceDeletedEvent < EventStore::Event
+  default_scope -> { where(event_type: ::Services::ServiceDeletedEvent) }
+
+  scope :by_object_id, ->(service_id) { where("data LIKE ('%service_id: #{service_id},%')") }
```

Comment:
@Martouta just reminded me that it's stored as a YAML, not as a JSON. So we need to replace `,%` with a `\n%`.